frontend/src/components/dashboard/create/govsg/GovsgRecipients.tsx

```diff
@@ -184,7 +184,7 @@ const GovsgRecipients = ({
           <FileInput isProcessing={isUploading} onFileSelected={uploadFile} />
           <p>or</p>
           <SampleCsv
-            params={params}
+            params={['language', ...params]}
```

Comment:
I didn't want to put `language` as a fixed header inside `SampleCsv` because `SampleCsv` is used beyond Gov.sg.